compiler/qsc_frontend/src/typeck/tests.rs

```diff
@@ -4415,3 +4440,236 @@ fn within_block_should_be_unit_error() {
         "##]],
     );
 }
+
+#[test]
+fn path_field_access() {
+    check(
+        indoc! {"
+            namespace A {
+                struct B { C : Int }
+                function Foo() : Unit {
+                    let b = new B { C = 5 };
+                    b.C;
+                }
+            }
+        "},
+        "",
+        &expect![[r##"
```

Comment:
nit: These double hashes are creeping up again 🙁

Reply:
@swernli can you clarify what we should be doing with new tests w/r/t double hashes?

From the `#![allow(clippy::needless_raw_string_hashes)]` suppression at the top of this file, and the `##`s in many other test cases, I'm led to believe we're supposed to _keep_ the double hashes as `expect-test` generates them.

Reply:
I don't think we should worry about these. If you manually remove them, the next time the expect-test updates they will get added back. The point of suppressing the warning is not fight that quirk of behavior, so I think we should ignore these differences. The only exception is when it grows each time (two become three, next run makes three become four), but I think that bug in expect-test was fixed so the problem doesn't occur anymore.